components/fires-server/.env.example

```diff
@@ -8,14 +8,12 @@ PUBLIC_URL=http://$HOST:$PORT/
 LOG_LEVEL=info
 
 # You can generate an APP_KEY with: node ace generate:key
-APP_KEY=
+# APP_KEY=
```

Comment:
This is commented to ensure that people do actually generate a `APP_KEY` using `node ace generate:key`